src/utils/requireAWS.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable global-require */
+const AWSXRay = require('aws-xray-sdk');
+const config = require('../config');
+
+let AWS;
+
+if (config.clusterEnv === 'test' || config.clusterEnv === 'development' || process.env.JEST_WORKER_ID) {
+  AWS = require('aws-sdk');
+} else {
+  AWS = AWSXRay.captureAWS(require('aws-sdk'));
+}
+
+module.exports = AWS;
```

Comment:
XRay can automatically hook into AWS services and log the calls made to them. For this to work reliably, I created a new helper, `requireAWS`. It will capture AWS calls in staging and production.